cmd/preflight/cmd/check_container.go

```diff
@@ -211,3 +254,75 @@ func generateContainerCheckOptions(cfg *runtime.Config) []container.Option {
 
 	return o
 }
+
+// artifactsTar takes a source path and a writer; a tar writer loops over the files in the source
+// directory, writes the appropriate header information and copies the file into the tar writer
+//
+//nolint:unparam // ctx is unused. Keep for future use.
+func artifactsTar(ctx context.Context, src string, w io.Writer) error {
```

Comment:
I haven't tested this here, but I think you can remove the nolint directive by anonymous assigning `ctx`. E.g.

```suggestion
func artifactsTar(_ context.Context, src string, w io.Writer) error {
```

Reply:
That does pass a lint check, but I think I'd rather keep the `nolint` since in an IDE the `_` makes the code look like it's used, ie highlighted vs grey'd out. But if other's prefer `_` I'll switch to that.

Reply:
I do prefer the `_` here, I think it's the appropriate application. Happy to have someone else weigh in.

Reply:
I'm on the fence here. If you go with the _, I'd still leave the //nolint:unparam (or just the comment) so it's clear to future someone why that's even there.

Reply:
With this in mind, up to you @acornett21 how you'd like to proceed.

Reply:
For consistency with other functions that do not use context, I'll keep this the as is in this PR.